### PR TITLE
Fixes Ghetto Grinding Cigarettes

### LINKED ISSUE #83947: cant crush cigarette in a mortar
Reporting client version: 515.1633

## Round ID:
[230482](https://scrubby.melonmesa.com/round/230482)

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)

## Reproduction:
But you can grind them in a grinder to get nicotine.

### PR DESCRIPTION
**Pull request by LucyGrind**
## About The Pull Request
fixes #83947
allowing cigarettes to be grinded in mortars again
## Why It's Good For The Game
Need muh nic sippy cup for my orny child
(fixing bugs is good for game health, duh)
## Changelog
:cl:
fix: allows cigarette grinding in mortar
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
You should probably specify that your fix is to _allow_ you to grind cigarettes, that changelog would be kind of misleading to read.

**Comment by LucyGrind** [comment]
> You should probably specify that your fix is to _allow_ you to grind cigarettes, that changelog would be kind of misleading to read.

donezo, thank you

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -286,6 +286,8 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	if(glass.reagents.trans_to(src, chem_volume, transferred_by = user)) //if reagents were transferred, show the message
 		to_chat(user, span_notice("You dip \the [src] into \the [glass]."))
 	//if not, either the beaker was empty, or the cigarette was full
+	if(istype(glass, /obj/item/reagent_containers/cup/mortar))
+		return NONE
```
Comment (Jacquerel) [inline code/game/objects/items/cigs_lighters.dm:290]: Are you sure this is the change you want?
I think this should probably be combined with the lines above, the check for "not being a reagent container"

The interaction _I think_ you have created is that you will "dip" the cigarette in whatever reagents are in the mortar and _then_ insert the cigarette into the mortar, but the user won't get any feedback that they did this.

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/game/objects/items/cigs_lighters.dm (hunk 1 above)
Comment (LucyGrind) [reply code/game/objects/items/cigs_lighters.dm:290]: if its above, then you cant dip the cigarette in a full mortar, which i didnt like, and if its half full, the cigarette gets covered in the mortar juice when you dip/insert it. say you inserted a cig in a mortar full of plasma, then removed it. It should be covered in plasma, no? This covers all of that

Comment (LucyGrind) [reply code/game/objects/items/cigs_lighters.dm]: fixed

**Pull request merged**